src/auto-evo/steps/FindBestMutation.cs

```diff
@@ -17,12 +17,16 @@ public class FindBestMutation : VariantTryingStep
 
         private readonly Mutations mutations = new();
 
-        public FindBestMutation(AutoEvoConfiguration configuration, PatchMap map, Species species,
+        private GameWorld gameWorld;
+
+        public FindBestMutation(AutoEvoConfiguration configuration,
```

Comment:
It seems that the game world is used pretty often so maybe AutoEvoConfiguration should have the world as a mandatory constructor parameter so it'd be available everywhere in auto-evo or maybe at least the world settings should be in there. As only needing world settings will be easier to use for auto-evo prediction and the auto-evo exploring tool, I imagine, when that is done.

Reply:
You are right, I was thinking of `SimulationConfiguration` which doesn't seem to be passed as a parameter here so the data can't be added there to easily use here.